packages/conf/src/migrations/normalizeRuntimeConfigModule.ts

```diff
@@ -0,0 +1,41 @@
+import { LinguiConfig, LinguiConfigNormalized } from "../types"
+
+type ModuleSrc = [source: string, identifier?: string]
+
+const getSymbolSource = (
+  defaults: ModuleSrc,
+  config: LinguiConfig["runtimeConfigModule"]
```

Comment:
Maybe it would be better to name this variable `runtimeConfigModule` instead of `config` as only the `runtimeConfigModule` property is passed to this function and not the whole config.